conans/client/rest/rest_client_v1.py

```diff
@@ -142,9 +142,7 @@ def _upload_files(self, file_urls, files, output, retry, retry_wait):
         t1 = time.time()
         failed = []
         uploader = Uploader(self.requester, output, self.verify_ssl)
-        # Take advantage of filenames ordering, so that conan_package.tgz and conan_export.tgz
-        # can be < conanfile, conaninfo, and sent always the last, so smaller files go first
-        for filename, resource_url in sorted(file_urls.items(), reverse=True):
+        for filename, resource_url in sorted(file_urls.items()):
```

Comment:
So comment here what is the advantage of the regular "sort" order (without reverse)